<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>control_window</class>
 <widget class="QMainWindow" name="control_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>198</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#startButton	
{
   background-image: url(:/newPrefix/play_pause.png);
   border-style: none;
}
QPushButton#startButton:hover
{
   background-image:url(:/newPrefix/play_pause_red.png);
   border-style: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0)</string>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(43, 255, 92);</string>
    </property>
    <property name="text">
     <string>Elapsed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 0, 0);</string>
    </property>
    <property name="text">
     <string>Remaining</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>H:MM:SS</string>
    </property>
   </widget>
   <widget class="QListWidget" name="history_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(80, 91, 104);
	border-style: none;
}</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="hr_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox
            {
            	color: rgb(140,140,140);
            	background-color: rgb(56, 56, 56);
				border-style:none;
            }
QSpinBox::up-button
            {

            }

QSpinBox::up-arrow
            {

            }
QSpinBox::down-button
            {

            }

QSpinBox::down-arrow
            {

            }</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="clock_text">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0)</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="elapsed_text">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(43, 255, 92);</string>
    </property>
    <property name="text">
     <string>0:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="remaining_text">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 0, 0);</string>
    </property>
    <property name="text">
     <string>0:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#exitButton
{
   background-color: rgb(56, 56, 56);
   color: rgb(197, 197, 197);
   border-style: none;
}
QPushButton#exitButton:hover
{
   background-color: rgb(155, 83, 83);
   color: rgb(197, 197, 197);
   border-style: none;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_box">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox
            {
            	color: rgb(140,140,140);
            	background-color: rgb(56, 56, 56);
				border-style:none;
            }
QSpinBox::up-button
            {

            }

QSpinBox::up-arrow
            {

            }
QSpinBox::down-button
            {

            }

QSpinBox::down-arrow
            {

            }</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sec_box">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox
            {
            	color: rgb(140,140,140);
            	background-color: rgb(56, 56, 56);
				border-style:none;
            }
QSpinBox::up-button
            {

            }

QSpinBox::up-arrow
            {

            }
QSpinBox::down-button
            {

            }

QSpinBox::down-arrow
            {

            }</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>196</y>
      <width>33</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton#resetButton	
{
   background-image: url(:/newPrefix/reset.png);
   border-style: none;
}
QPushButton#resetButton:hover
{
   background-image: url(:/newPrefix/reset_red.png);
   border-style: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fullscreenButton">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>198</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#fullscreenButton	
{
   background-image: url(:/newPrefix/fullscreen.png);
   border-style: none;
}
QPushButton#fullscreenButton:hover
{
   background-image: url(:/newPrefix/fullscreen_red.png);
   border-style: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>59</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/countdown_logo.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>200</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#deleteButton	
{
   background-image: url(:/newPrefix/delete.png);
   border-style: none;
}
QPushButton#deleteButton:hover
{
   background-image: url(:/newPrefix/delete_red.png);
   border-style: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="countdown_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
